src/urllib3/poolmanager.py

```diff
@@ -294,6 +294,8 @@ def connection_from_host(
             raise LocationValueError("No host specified.")
 
         request_context = self._merge_pool_kwargs(pool_kwargs)
+        if scheme is None:
```

Comment:
Almost no one uses urllib3's `connection_from_context()` method directly, so I think we're safe to hoist the warning to come from `urlopen` so that the `stacklevel` isn't a strange looking function.

Reply:
Done ✔️